package.json

```diff
@@ -55,7 +56,5 @@
     "typescript": "^5.5.4"
   },
   "prettier": "@spotify/prettier-config",
-  "dependencies": {
-    "@changesets/cli": "^2.18.0"
-  }
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
this is being added automatically because I'm using corepack locally. It shouldn't hurt anything and in theory gives you more consistency across contributors.